contracts/fast/FastTokenFacet.sol

```diff
@@ -403,4 +408,28 @@ contract FastTokenFacet is AFastFacet, IERC20, IERC1404 {
     require(a != b, LibConstants.REQUIRES_DIFFERENT_SENDER_AND_RECIPIENT);
     _;
   }
+
+  /** @dev Ensures that the given address is a member of the current FAST or the Zero Address.
```

Comment:
I decided to move this modifier here, because it's very unlikely that any other facet will need it - so it decouples dependencies between `AFastFacet` and the exchange etc. This will solve some problems that I have implementing the "deactivated" user feature on the exchange.